packages/utils/src/customer-requests/full-snomed-pipeline.ts

```
@@ -0,0 +1,114 @@
+// This script is used to process all of a customers patients through a snomed filtering pipeline, and
+// then generate MR summaries and CSV files from the resulting bundles
+// The script makes consolidated queries, inserts a patient FHIR resource, and then calls the filtering
+// and MR generation logic.
+
+import { bundleToHtml } from "@metriport/core/external/aws/lambda-logic/bundle-to-html-snomed";
+import fs from "fs/promises";
+import { convertHtmlTablesToCsv } from "./convert-html-to-csv";
+import * as path from "path";
+import { fullProcessing } from "../terminology-server/snomed-dedup-and-filter";
+import { MetriportMedicalApi } from "@metriport/api-sdk";
+import { getEnvVarOrFail } from "@metriport/core/util/env-var";
+import * as dotenv from "dotenv";
+import axios from "axios";
+import dayjs from "dayjs";
+import duration from "dayjs/plugin/duration";
+dotenv.config();
+dayjs.extend(duration);
+
+const apiKey = getEnvVarOrFail("API_KEY");
+const apiLoadBalancerURL = getEnvVarOrFail("API_URL");
+const fhirUrl = getEnvVarOrFail("FHIR_URL");
+const cxId = getEnvVarOrFail("CX_ID");
+const metriportApi: MetriportMedicalApi = new MetriportMedicalApi(apiKey);
+
+const timestamp = dayjs().toISOString();
+const resultsDirectory = `./runs/consolidatedPatients/${timestamp}`;
+
+async function fetchPatientIds(cxId: string): Promise<string[]> {
+  const url = `${apiLoadBalancerURL}/internal/patient/ids?cxId=${cxId}`;
+  try {
+    const response = await axios.get(url);
+    return response.data.patientIds;
+  } catch (error) {
+    console.error(`Error fetching patient IDs:`, error);
+    return [];
+  }
+}
+
+//eslint-disable-next-line
+async function getFhirPatientData(patientId: string): Promise<any> {
+  const url = `${fhirUrl}/fhir/${cxId}/Patient/${patientId}/`;
+  try {
+    const response = await axios.get(url);
+    return response.data;
+  } catch (error) {
+    console.error(`Error fetching FHIR data for patient ${patientId}:`, error);
+  }
```

This is failing silently and the code will keep executing. Is that expected?